pkg/transform/transform.go

```diff
@@ -84,6 +84,7 @@ func Start() {
 
 	runner.Transform([]Transform{
 		APITransform{},
+		ClusterReportTransform{},
```

Comment:
Not a biggie, but what about using just `ClusterTransform{}` although it does produce only report it still goes through the ETL flow ?